packages/jest-core/src/cli/index.ts

```diff
@@ -89,7 +89,9 @@ export async function runCLI(
     configsOfProjectsToRun,
```

Comment:
glad to see my `_run10000` is still in and no one felt the need to revert it :D

Reply:
easter egg 😀